yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -424,7 +446,11 @@ func gatherAssessmentMetadataFromPG() (err error) {
 	return nil
 }
 
-func parseExportedSchemaFileForAssessment() {
+func parseExportedSchemaFileForAssessmentIfRequired() {
+	if source.DBType == ORACLE {
```

Comment:
sure, will add comment in the code. But overall this is due to difference in working of pg_dump and ora2pg.
In pg_dump, we ourself divide the schema in separate `.sql` files
While in case of oracle, we export the object types one by one, in their respective folder/files itself. 

So for PG, i need to separate out the logic of export schema and then categorising them into `.sql` files so that assess migration can still run in case of no source db connectivity through voyager